src/fsdocs-tool/BuildCommand.fs

```diff
@@ -550,40 +556,64 @@ type internal DocContent
                          Int32.MaxValue)
                 | None -> Int32.MaxValue)
 
-        [
-          // No categories specified
-          if modelsByCategory.Length = 1 && (fst modelsByCategory.[0]) = None then
-              li [ Class "nav-header" ] [ !! "Documentation" ]
-
-              for model in snd modelsByCategory.[0] do
-                  let link = model.Uri(root)
-
-                  li [ Class "nav-item" ] [ a [ Class "nav-link"; (Href link) ] [ encode model.Title ] ]
-          else
-              // At least one category has been specified. Sort each category by index and emit
-              // Use 'Other' as a header for uncategorised things
-              for (cat, modelsInCategory) in modelsByCategory do
-                  let modelsInCategory =
-                      modelsInCategory
-                      |> List.sortBy (fun model ->
-                          match model.Index with
-                          | Some s ->
-                              (try
-                                  int32 s
-                               with _ ->
-                                   Int32.MaxValue)
-                          | None -> Int32.MaxValue)
-
-                  match cat with
-                  | Some c -> li [ Class "nav-header" ] [ !!c ]
-                  | None -> li [ Class "nav-header" ] [ !! "Other" ]
-
-                  for model in modelsInCategory do
+        if FSharp.Formatting.Common.Menu.isTemplatingAvailable input then
+            let createGroup (header: string) (items: LiterateDocModel list) : string =
+                //convert items into menuitem list
+                let menuItems =
+                    items
```

Comment:
The sort order of the menu items is not yet respected.

![image](https://user-images.githubusercontent.com/2621499/184296635-e8a10262-53a3-43ab-b713-cca35d8cb497.png)

This is alphabetical now, the original code sorts it like this:
```fsharp
|> List.sortBy (fun model ->
                              match model.Index with
                              | Some s ->
                                  (try
                                      int32 s
                                   with _ ->
                                       Int32.MaxValue)
                              | None -> Int32.MaxValue)
```

Extract the existing sort to a local helper function and re-use it in the new code.